apps/events/urls.py

```diff
@@ -1,17 +1,14 @@
 from django.urls import path, include
 from rest_framework import routers
 
-from .viewsets import (
-    EventViewSet,
-    RatingViewSet,
-    TagViewSet,
-    MarkerViewSet,
-    CategoryViewSet,
-    ReviewViewSet,
-    CurrencyViewSet,
-    ParticipantViewSet
-)
-
+from .viewsets.events import EventViewSet
+from .viewsets.ratings import RatingViewSet
+from .viewsets.tags import TagViewSet
+from .viewsets.markers import MarkerViewSet
+from .viewsets.categories import CategoryViewSet
+from .viewsets.reviews import ReviewViewSet
+from .viewsets.currencies import CurrencyViewSet
+from .viewsets.participants import ParticipantViewSet
```

Comment:
It's better to have a nice single import like it was previously. To do that you can just use __init__.py inside viewsets and use __all__, example can be seen in events/serializers/__init__.py